afkak/brokerclient.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 # Copyright 2015 Cyan, Inc.
-# Copyright 2017, 2018, 2019 Ciena Corporation
+# Copyright 2017, 2018, 2019, 2021 Ciena Corporation
```

Comment:
Python 2 reference in comment at [line 83](https://github.com/ciena/afkak/pull/118/files#diff-cd67ccf60d169cdb5b24c4ef85d6a1c93f22996af62892d13c1933e2dc702eabR83).